instruction/db-sql/db-sql.md

```diff
@@ -65,7 +65,10 @@ All databases have a default character set that is used for representing bytes a
 ALTER DATABASE pet_store CHARACTER SET utf8mb4;
 ```
 
-If you want to delete a database, and all of the data it represents, you use the `DROP DATABASE` statement. Be careful with this statement. Once you run it there is no going back. The data is gone forever.
+If you want to delete a database, and all of the data it represents, you use the `DROP DATABASE` statement. While this should rarely be used in a production environment, it is common to drop databases in development environments so that changes to tables can be observed (see ["Changing Tables"](#changing-tables) for more info.)
+
+> [!CAUTION]
```

Comment:
I am a bit worried that we are over using the TIP, IMPORTANT feature. If used too much it becomes a distraction. Is this warning really important? It seems pretty obvious and has little danger with what they are doing since there is no "production" data to lose.

Same thing with the CAUTION under Changing tables